public/js/homepage.js

```diff
@@ -0,0 +1,29 @@
+// btn-hence working on service button carousel
+
+$('.btn-hence').owlCarousel({
+    loop: true,
+    margin: 0,
+    nav: false,
+    dots: false,
+    autoplay: true,
+    responsive: {
+        0: {items: 2},
+        600: {items: 3},
+        1000: {items: 4}
+    }
+});
+
+// testimonial carousel
+
+$('.testimonial').owlCarousel({
```

Comment:
if this is also a css class then do not use it in js identifier

Reply:
i provide class name which is only identifier